data/tutorials/gs_00_up_and_running.md

```diff
@@ -13,8 +13,9 @@ date: 2021-05-27T21:07:30-00:00
 
 This page will walk you through the installation of everything you need for a comfortable development environment to write projects in OCaml code. Of course, this includes [installing the compiler](#installing-ocaml) itself, but it also installs a build system, a package manager, an LSP server to support your editor, and a few other tools that we describe [later](#setting-up-development-tools), setting up [editor support](#configuring-your-editor), and bootstrapping a [new project](#starting-a-new-project).
 
-The following instructions work on Linux, BSD, and macOS. Plus, they also work
-on Cygwin and WSL. If you want to set up OCaml on Windows, you might be
+We provide installation instructions for Unix-derivatives - including Linux, BSD, and macOS - and for Windows via the [Diskuv OCaml](https://github.com/diskuv/dkml-installer-ocaml#readme) Installer. The Unix instructions can be used on Windows if you use Windows Subsystem for Linux (WSL).
```

Comment:
nit: "BSD" is not an operating system, as FreeBSD, NetBSD and OpenBSD are all distinct these days.  It might be simpler to say "We provide installation instructions for Linux, *BSD, macOS and Windows via the ...".

Also, the Diskuv installer doesn't work on OCaml 5.0 does it? (or does it work for that for everything but Windows?)

Reply:
Good point about the *BSDs. I'll change that.

The Diskuv Installer seems to be the simplest way to install on Windows, and indeed, the page mentions it's only for 4.14.0 - when people need a different version, they unfortunately need to go through the more comprehensive instructions. When there is a better "simplest way to install OCaml on Windows", we should update this page.